packages/language-server-bitbake/package.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: language-server-bitbake
+description: Language Server for Bitbake.
+homepage: https://github.com/yoctoproject/vscode-bitbake
+licenses:
+  - MIT
+languages:
+  - Bitbake
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/language-server-bitbake@2.5.0
+
+schemas:
+  lsp: vscode:https://raw.githubusercontent.com/yoctoproject/vscode-bitbake/v{{version}}/server/package.json
```

Comment:
I'm not sure whether this line should point to the client's or the server's `package.json`. Both are available under `vscode-bitbake`. The client there is a vscode client.